jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/resources/META-INF/services/org.jbpm.persistence.api.integration.EventEmitter

```diff
@@ -0,0 +1 @@
+org.jbpm.event.emitters.kafka.KafkaEventEmitter
```

Comment:
for testing ok, for prod not. We can only have one emitter. You have two options here:
1) Either you add some logic to set manually the emitter.
2) You remove this.